code/game/machinery/teleporter.dm

```diff
@@ -74,8 +74,12 @@
 				if(ishuman(M))//don't remove people from the round randomly you jerks
 					var/mob/living/carbon/human/human = M
 					if(human.dna && human.dna.species.id == "human")
-						to_chat(M, "<span class='italics'>You hear a buzzing in your ears.</span>")
-						human.set_species(/datum/species/fly)
+						if(prob(50))
+							to_chat(M, "<span class='italics'>You hear a buzzing in your ears.</span>")
+							human.set_species(/datum/species/fly)
+						else
+							to_chat(M, "<span class='italics'>You hear fluttering in your ears.</span>")
```

Comment:
what justification do you have for this?

If they're going to be added it'll be a server decision to enable them as a race

Reply:
the whole fly thing is a backported reference and isn't subject to the same rules